os/booting-with-libvirt.md

```diff
@@ -82,7 +82,7 @@ ssh_authorized_keys:
  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDGdByTgSVHq.......
 ```
 
-Note: The `$private_ipv4` and `$public_ipv4` substitution variables referenced in other documents are *not* supported on libvirt.
+Note: The `$private_ipv4` and `$public_ipv4` substitution variables referenced in other documents are *not* supported on libvirt. But you can host your Cloud-Config using nginx and automatically sunstitute these variables following [nginx][nginx] doc.
```

Comment:
Note: The `$private_ipv4` and `$public_ipv4` substitution variables referenced in other documents are *not* supported on libvirt. The convenience of these automatic variables can be emulated by using nginx to host your cloud-config, as described in [Nginx Cloud-config Hosting][nginx]. 

(repeat in other locations)